betterforms/multiform.py

```diff
@@ -73,13 +75,32 @@ def __iter__(self):
     def is_bound(self):
         return any(form.is_bound for form in self.forms.values())
 
+    def clean(self):
+        """
+        Raises any ValidationErrors required for cross form validation. Should
+        return a dict of cleaned_data objects for any forms whose data should
+        be overridden.
+        """
+        return self.cleaned_data
+
+    def add_crossform_error(self, e):
+        self.crossform_errors.append(e)
+
     def is_valid(self):
-        return all(form.is_valid() for form in self.forms.values())
+        forms_valid = all(form.is_valid() for form in self.forms.values())
+        try:
+            cleaned_data = self.clean()
+        except ValidationError as e:
+            self.add_crossform_error(e)
+        else:
+            if cleaned_data is not None:
+                for key, data in cleaned_data.items():
+                    self.forms[key].cleaned_data = data
```

Comment:
If you remove this code, I believe the tests will still pass, but then we will have errors.